resources/views/auth/register.blade.php

```diff
@@ -115,8 +114,8 @@
                                 </span>
                             @endif
                             </div>
-
                         </div>
+                        -->
```

Comment:
this also

Reply:
Changed in line with the latest Wire Frame.